src/services/EnvironmentVariables.ts

```diff
@@ -304,6 +306,15 @@ export class EnvironmentVariables {
 				break;
 			}
 			
+			case ServicesEnum.GENERATE_YOTI_LETTER_SERVICE: {
```

Comment:
I can see a couple more env vars being used for this new lambda, seem to be missing here? Also the logs dont list out all the vars.